### ISSUE TITLE
[Bug] Pan canvas on token after teleport

### ISSUE THREAD

**Issue by adotor** [grandseiken/foundryvtt-multilevel-tokens#135]
After teleporting the canvas does not pan onto the teleported token. Looking at HowTo videos it looks like this was a feature. I found no option to toggle this.

Foundry: 10.290
System: DnD5e 2.0.3
Multilevel Token: 1.6.0

**Comment by DrMcCoy** [comment]
Yeah, it seems that this is one thing that got lost in Foundry VTT v10. Found this out in a recent session when my players complained they didn't know where they got teleported to, when that worked previously with v9.

**Comment by MB1997** [comment]
I'm having this issue, too. Its not a critical bug for me, since i have everything on 1 map, but its far from player friendly. 
Hope to get it fixed soon.

**Comment by funwithfondue** [comment]
+1 for this issue getting fixed. Not a huge issue but would make the module more user friendly.
